sql-queries/sql-column-name/sql-column-name-postgresqlSELECT column_name FROM information_schema.columns WHERE table_name = 'Course';.sql

```diff
@@ -0,0 +1,5 @@
+SELECT column_name
```

Comment:
I don't think this is needed for this tutorial, can you remove it from the PR, please?

Reply:
You can remove this from the PR by deleting the file on your filesystem and then running

```
git add .
git push
```